frameworks/Scala/zio-http/src/main/scala/Main.scala

```diff
@@ -0,0 +1,41 @@
+import zhttp.http._
+import zhttp.service.Server
+import zio.{App, ExitCode, URIO}
+import io.netty.handler.codec.http.{HttpHeaderNames => JHttpHeaderNames, HttpHeaderValues => JHttpHeaderValues}
+
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+
+object WebApp extends App {
+
+  val message: String = "Hello, World!"
+def createDate: String = DateTimeFormatter.RFC_1123_DATE_TIME.format(ZonedDateTime.now)
+  import com.github.plokhotnyuk.jsoniter_scala.macros._
+  import com.github.plokhotnyuk.jsoniter_scala.core._
+
+  case class Message(message: String)
+  implicit val codec: JsonValueCodec[Message] = JsonCodecMaker.make
+
+  val jsonstr = new String(writeToArray(Message(message)))
```

Comment:
Maybe it's just me, but I'd say manually encoding the value to a byte array defeats the purpose of testing the framework's performance, since this code is part of the test and not the framework.

Reply:
We wanted to not have circe or jsoniter dependency in zio-http. We are reevaluating this will get back